shotover-proxy/src/transforms/cassandra/sink_cluster/node_pool.rs

```diff
@@ -73,6 +73,22 @@ impl NodePool {
         write_lock.insert(id, metadata);
     }
 
+    pub fn get_shuffled_nodes_in_dc_rack(
```

Comment:
*shuffled addresses, we say nodes in the other methods when returning a `CassandraNode`

Reply:
done